app/team.go

```diff
@@ -161,12 +161,23 @@ func (a *App) SoftDeleteAllTeamsExcept(teamID string) *model.AppError {
 	return nil
 }
 
-func shouldCreateOnboardingLinkedBoard(a *App, c request.CTX, teamId string) bool {
+func (a *App) shouldCreateOnboardingLinkedBoard(c request.CTX, teamId string) bool {
 	ffEnabled := a.Config().FeatureFlags.OnboardingAutoShowLinkedBoard
 	if !ffEnabled {
 		return false
 	}
 
+	hasBoard, err := a.HasBoardProduct()
+	if err != nil {
+		mlog.Error("Error checking the existance of boards product: ", mlog.Err(err))
```

Comment:
2 things: 
- in `App` you should use `a.Log().Error(...)` instead of using `mlog.Error` directly.
- we usually don't use uppercase for logged messages. 
- the `: ` at the end is not necessary: the first argument is your message, the error itself is just gonna be an "attribute" of the log and can be anywhere in the final log string. For example the output for this could be `{"date": "xxxxx", "msg": "Error checking the existance of boards product: ", "mm-version": "yyyy", "err": "whatever is in the err variable"}`

Reply:
👍